Makefile

```diff
@@ -760,7 +760,8 @@ else
 ifeq ($(TARGET),$(filter $(TARGET),$(F1_TARGETS)))
 OPTIMIZE    = -Os
 else
-OPTIMIZE    = -Ofast
+#OPTIMIZE    = -Ofast
+OPTIMIZE    = -O2
```

Comment:
This doesn't seem to be justified just for this update. #1747 should provide relief for this. Please stand by...